### ISSUE TITLE
Investigate `lineOffset` argument to `compile`; figure out if it is implemented correctly

### ISSUE THREAD

**Issue by cspotcode** [TypeStrong/ts-node#1139]
Our `compile` function accepts a third argument, `lineOffset`

The REPL passes a value, but the implementation ignores that argument.

Should we be using it for something?  Is it meant to modify error line numbers so that REPL errors are always starting at line 1?

I suspect this may be something that I broke, erroneously assuming that it was not important.

@blakeembrey do you remember?

**Comment by blakeembrey** [comment]
You're correct for the intended usage, though I don't think you broke it. I feel like I broke it when trying to refactor for TypeScript incremental compilation or something. Not sure how easy it is to add back everywhere.
